src/main/java/io/gravitee/resource/authprovider/ldap/LdapAuthenticationProviderResource.java

```diff
@@ -90,31 +95,32 @@ public void authenticate(String username, String password, ExecutionContext cont
     protected void doStart() throws Exception {
         super.doStart();
 
-        pooledConnectionFactory = bindPooledConnectionFactory();
-        searchPooledConnectionFactory = searchPooledConnectionFactory();
+        pooledConnectionFactory = getPooledFactory();
+        searchPooledConnectionFactory = getPooledFactory();
```

Comment:
Does it make sense to get twice the same pooled factory ?

Reply:
comments from gravitee-am-identityprovider-ldap: 
 /**
     * Bind OPERATIONS configuration
     * We must split the connections for search and bind operations because bind operation change the connection context to the last user authenticated
     * If the user has no role to search in the LDAP/AD directory the authentication will fail for the next users
     */

    /**
     * Search OPERATIONS configuration
     * We must split the connections for search and bind operations because bind operation change the connection context to the last user authenticated
     * If the user has no role to search in the LDAP/AD directory the authentication will fail for the next users
     */

Reply:
I understand it but previsouly both pool had different configuration where the SearchConnectionValidator was only applied on the search pool. Here you create the exact same one twice.

Reply:
bind had SearchValidator
search had SearchConnectionValidator

in ldaptive2 SearchValidator does not exist and I can use only SearchConnectionValidator. In that case, the configuration is the same.